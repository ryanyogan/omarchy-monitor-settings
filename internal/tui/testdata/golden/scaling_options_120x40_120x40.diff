Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m│[0m                                                [1;34mDisplay Settings[0m                                                [90m│[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                        
  [32m╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m  [1;32m🧠 Smart Scaling Recommendations[0m                                                                              [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m  [33m╭─────────────────────────────────────────╮[0m                                                                   [32m│[0m    
  [32m│[0m  [33m│[0m[40m                                         [0m[33m│[0m                                                                   [32m│[0m    
  [32m│[0m  [33m│[0m[40m  [0m[40mHDMI-A-1 Dell U2414H - 1920x1080@60Hz[0m[40m  [0m[33m│[0m                                                                   [32m│[0m    
  [32m│[0m  [33m│[0m[40m                                         [0m[33m│[0m                                                                   [32m│[0m    
  [32m│[0m  [33m╰─────────────────────────────────────────╯[0m                                                                   [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m  [1;36m🎯 Available Options[0m                                                                                          [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m  [1;34m▶ [0m[1;34m1x Native [RECOMMENDED][0m                                                                                     [32m│[0m    
  [32m│[0m  [37m    Native resolution with standard scaling[0m                                                                   [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080[0m                                                [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                                                                       [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m    1.25x Enhanced                                                                                              [32m│[0m    
  [32m│[0m  [37m    Slightly larger text for better readability[0m                                                               [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864[0m                                                [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                                                                       [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m    1.5x Large                                                                                                  [32m│[0m    
  [32m│[0m  [37m    Larger text for accessibility[0m                                                                             [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.5x • GTK: 1x • Font DPI: 144 • Result: 1280x720[0m                                                [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                                                                       [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m  [1;35m📚 What Each Setting Does[0m                                                                                     [32m│[0m    
  [32m│[0m  [37m  Monitor Scale: Changes compositor-level scaling (immediate effect)[0m                                          [32m│[0m    
  [32m│[0m  [37m  GTK Scale: Scales GTK applications (requires logout/login)[0m                                                  [32m│[0m    
  [32m│[0m  [37m  Font DPI: Fine-grained text scaling (affects most apps)[0m                                                     [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m  [32m↑↓[0m[37m select[0m  [34m⏎[0m[37m apply[0m  [33mm[0m[37m manual[0m  [35mesc[0m[37m back[0m                                                                        [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m│[0m                                                                                                                [32m│[0m    
  [32m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                        
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m│[0m                           [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m                         [90m│[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                Display Settings                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                                                                                │    
  │  🧠 Smart Scaling Recommendations                                                                              │    
  │                                                                                                                │    
  │  ╭─────────────────────────────────────────╮                                                                   │    
  │  │                                         │                                                                   │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz  │                                                                   │    
  │  │                                         │                                                                   │    
  │  ╰─────────────────────────────────────────╯                                                                   │    
  │                                                                                                                │    
  │  🎯 Available Options                                                                                          │    
  │                                                                                                                │    
  │  ▶ 1x Native [RECOMMENDED]                                                                                     │    
  │      Native resolution with standard scaling                                                                   │    
  │      Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080                                                │    
  │      💡                                                                                                        │    
  │                                                                                                                │    
  │    1.25x Enhanced                                                                                              │    
  │      Slightly larger text for better readability                                                               │    
  │      Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864                                                │    
  │      💡                                                                                                        │    
  │                                                                                                                │    
  │    1.5x Large                                                                                                  │    
  │      Larger text for accessibility                                                                             │    
  │      Monitor: 1.5x • GTK: 1x • Font DPI: 144 • Result: 1280x720                                                │    
  │      💡                                                                                                        │    
  │                                                                                                                │    
  │                                                                                                                │    
  │  📚 What Each Setting Does                                                                                     │    
  │    Monitor Scale: Changes compositor-level scaling (immediate effect)                                          │    
  │    GTK Scale: Scales GTK applications (requires logout/login)                                                  │    
  │    Font DPI: Fine-grained text scaling (affects most apps)                                                     │    
  │                                                                                                                │    
  │  ↑↓ select  ⏎ apply  m manual  esc back                                                                        │    
  │                                                                                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                           ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                         │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 9092
Actual length: 7813
Expected hash: b8e8d995d9b6cbdbe2d7e57e255444fe8418cd2109199d56374b0c8c2ab5c131
Actual hash: 608f425e5028396173c54a8f4818dcab3c5c31da4f8dc693a4f08adf80c8eb87
