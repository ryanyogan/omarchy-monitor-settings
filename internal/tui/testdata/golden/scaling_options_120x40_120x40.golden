# Visual Golden File
# Name: scaling_options_120x40
# Dimensions: 120x40
# Hash: 608f425e5028396173c54a8f4818dcab3c5c31da4f8dc693a4f08adf80c8eb87

  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                Display Settings                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                                                                                │    
  │  🧠 Smart Scaling Recommendations                                                                              │    
  │                                                                                                                │    
  │  ╭─────────────────────────────────────────╮                                                                   │    
  │  │                                         │                                                                   │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz  │                                                                   │    
  │  │                                         │                                                                   │    
  │  ╰─────────────────────────────────────────╯                                                                   │    
  │                                                                                                                │    
  │  🎯 Available Options                                                                                          │    
  │                                                                                                                │    
  │  ▶ 1x Native [RECOMMENDED]                                                                                     │    
  │      Native resolution with standard scaling                                                                   │    
  │      Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080                                                │    
  │      💡                                                                                                        │    
  │                                                                                                                │    
  │    1.25x Enhanced                                                                                              │    
  │      Slightly larger text for better readability                                                               │    
  │      Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864                                                │    
  │      💡                                                                                                        │    
  │                                                                                                                │    
  │    1.5x Large                                                                                                  │    
  │      Larger text for accessibility                                                                             │    
  │      Monitor: 1.5x • GTK: 1x • Font DPI: 144 • Result: 1280x720                                                │    
  │      💡                                                                                                        │    
  │                                                                                                                │    
  │                                                                                                                │    
  │  📚 What Each Setting Does                                                                                     │    
  │    Monitor Scale: Changes compositor-level scaling (immediate effect)                                          │    
  │    GTK Scale: Scales GTK applications (requires logout/login)                                                  │    
  │    Font DPI: Fine-grained text scaling (affects most apps)                                                     │    
  │                                                                                                                │    
  │  ↑↓ select  ⏎ apply  m manual  esc back                                                                        │    
  │                                                                                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                           ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                         │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    